<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QEMU VM Launcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Run virtual machine disk image ISO file.</string>
    </property>
    <property name="whatsThis">
     <string>Button for running virtual machine.</string>
    </property>
    <property name="text">
     <string>Run virtual machine</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_disk_path">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>471</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to virtual ISO image</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_memory">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2024</number>
    </property>
    <property name="maximum">
     <number>640000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RAM memory</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enable_audio">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable audio</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="display">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit_app">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show_help">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="enable_ethernet">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Ethernet (NAT)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
